.env.example

```
@@ -1 +1 @@
-TEST=testvalue
+HERETRANSITAPI=apiKey
```

snake case for better readability?

```suggestion
HERE_TRANSIT_API=apiKey
```